### PR TITLE
Kafka reporters in gobblin-metrics switch the name of the schema to t…

### PR DESCRIPTION
**Pull request by ibuenros**
…he appropriate topic before registering in schema registry.

### CODE REVIEW

**Comment by ibuenros** [comment]
@chavdar can you review?

**Comment by ibuenros** [comment]
@zliu41 can you review?

File: gobblin-metrics/src/main/java/gobblin/metrics/kafka/SchemaUtils.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ */
+public class SchemaUtils {
+
+  /**
+   * Copies the input {@link org.apache.avro.Schema} but changes the schema name.
+   * @param schema {@link org.apache.avro.Schema} to copy.
+   * @param newName name for the copied {@link org.apache.avro.Schema}.
+   * @return A {@link org.apache.avro.Schema} that is a copy of schema, but has the name newName.
+   */
+  public static Schema switchName(Schema schema, String newName) {
```
Comment (zliu41) [inline gobblin-metrics/src/main/java/gobblin/metrics/kafka/SchemaUtils.java]: You can put this method in `AvroUtils`. There are other schema-related methods in that class.

**Comment by ibuenros** [comment]
@zliu41 addressed comments.

**Comment by zliu41** [comment]
LGTM

**Pull request merged**